worldedit-bukkit/adapters/adapter-1_18/src/main/java/com/sk89q/worldedit/bukkit/adapter/ext/fawe/v1_18_R1/PaperweightFakePlayer.java

```diff
@@ -37,7 +37,11 @@
 import java.util.UUID;
 
 class PaperweightFakePlayer extends ServerPlayer {
-    private static final GameProfile FAKE_WORLDEDIT_PROFILE = new GameProfile(UUID.nameUUIDFromBytes("worldedit".getBytes()), "[WorldEdit]");
+
+    private static final GameProfile FAKE_WORLDEDIT_PROFILE = new GameProfile(
+            UUID.nameUUIDFromBytes("worldedit".getBytes()),
+            "[WorldEdit]"
+    );
```

Comment:
Why did this change to a multiline assignment? Is it because of our project code style? If it all fits on one line I would prefer to keep it that way.

Reply:
I'd assume it's slightly too long?

Reply:
The diff is the way how our line breaks look like